cmd/lightningstream/commands/snapshots.go

```diff
@@ -32,7 +33,7 @@ func init() {
 
 	snapshotsCmd.AddCommand(snapshotsDumpCmd)
 	snapshotsDumpCmd.Flags().StringP("format", "f", "debug",
-		"Output format, one of: 'debug' (default), 'text'")
+		"Output format, one of: 'debug' (default), 'text' (same)")
```

Comment:
The reason for this switch was a bit different.

In 0.3.0, you could use two output formats:

- 'text' is a google protobuf encoding that is reversible to binary, but it makes all the key-values unreadable
- 'debug' shows the all the key-value entries in a more friendly way

Now that we drop the generated protobuf code, we also loose the 'text' encoder'. I decided to just make this the same as 'debug', instead of dropping the 'text' option, because I figured it would be less likely to break anything.